lib/node_modules/@stdlib/math/base/special/fibonacci-index/benchmark/benchmark.js

```diff
@@ -37,14 +37,16 @@ bench( pkg, function benchmark( b ) {
 	var y;
 	var i;
 
-	FN = new Array( 76 );
+	FN = zeros( 76 );
```

Comment:
```suggestion
	FN = zeros( 79 );
```

Looks like a bug in the original.